certified-connectors/EBMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2582 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "EBMS",
+    "description": "Connect your EBMS data with hundreds of other services!  Contact your Eagle Business Software Account Manager to activate your Power Automate connector.",
```

Comment:
@doniyorniazov the description contains an extra whitespace, it also shouldn't contain `Power Automate` or any other Power Platform product names. One of the checks is failing because of this.